op-chain-ops/genesis/testdata/test-deploy-config-full.json

```diff
@@ -40,12 +40,15 @@
   "baseFeeVaultRecipient": "0x42000000000000000000000000000000000000f5",
   "l1FeeVaultRecipient": "0x42000000000000000000000000000000000000f6",
   "sequencerFeeVaultRecipient": "0x42000000000000000000000000000000000000f7",
+  "operatorFeeVaultRecipient": "0x42000000000000000000000000000000000000f8",
```

Comment:
should be able to delete per https://github.com/ethereum-optimism/optimism/pull/12166/files#r1941779240